annotation-client/src/test/java/eu/europeana/annotation/client/integration/webanno/tag/SemanticTaggingTest.java

```diff
@@ -85,4 +85,16 @@ public void createSemanticTagEntity() throws IOException, JsonParseException, Il
 		assertNotNull(agentBody.getPrefLabel());
 	}
 
+	@Test
+	public void createSemanticTagAgentEntity() throws IOException, JsonParseException, IllegalAccessException,
+			IllegalArgumentException, InvocationTargetException {
+
+		Annotation storedAnno = createTag(SEMANTICTAG_AGENT_ENTITY, false, true);
+		log.info(storedAnno.getBody().getInternalType());
+		assertTrue(storedAnno.getMotivation().equals(MotivationTypes.TAGGING.name().toLowerCase()));
+		assertEquals(storedAnno.getBody().getInternalType(), BodyInternalTypes.AGENT.name());
+		EdmAgent agentBody = (EdmAgent) ((EdmAgentBody) storedAnno.getBody()).getAgent();
```

Comment:
Do we support all fields from the given example?
We have to compare the body of the input with the body of the stored annotation similar to the the validateOutputAgainstInput(Annotation storedAnno, Annotation inputAnno)

Reply:
Entsprechend dem Ticket ist nur skos:prefLabel mandatory. I added additional fields to check for particular example.